target/scripts/startup/setup-stack.sh

```diff
@@ -1,5 +1,33 @@
 #!/bin/bash
 
+# shellcheck disable=SC2034
+declare -A VARS
```

Comment:
This was done due to a Bash having _very weird_ scoping. With #3115 and the follow-up, this is solved since `variables.sh` will become a proper "stack", so the sourcing will be done by `start-mailserver.sh`, which in turn makes declaring this array possible in `variables.sh` (then: `variables-stack.sh`).

Reply:
> This was done due to a Bash having _very weird_ scoping.

Yeah I figured that was the case :sweat_smile: 

I'm particularly interested in your ENV parser rust project having potential to eventually manage not only the ENV documentation, but share the same struct / module for what `variables.sh` is effectively responsible for.

Then we won't have drift, as recently experienced with the `SA_KILL` update to `mailserver.env`.

Reply:
I was actually thinking about extending the parser this way too :D